### DISCUSSION TITLE
How can I access a previous version of the docs?

### DISCUSSION THREAD

**Discussion by dknight10** [vercel/next.js#49494] [Help] [answered]
### Summary

Is there a way to see a previous version of the docs? I am using version 13.2 and the docs seem to be switched over to the new version so aren't very helpful to me. I don't see anything on the page that would allow switching to a different version.

**Comment by icyJoseph** [comment, marked as answer]
Hi,

I think you can only switch between pages and app router docs.

This doesn't work anymore: https://nextjs.org/docs/tag/v9.5.2/api-reference/next/link#dynamic-routes 😢 

I tried adding pages/docs but no luck either.

I guess you can try to navigate the repo docs: https://github.com/vercel/next.js/tree/v13.3.0/docs - kind of split on this one, the old docs should still be reachable somehow, at least down to next@12 I'd think.

**Reply by dknight10** [reply]
Yeah, I noticed that the tag/vx.x.x path to older versions doesn't work anymore. 

Honestly, I missed the (fairly obvious) toggle between the pages and app docs somehow. I think that should work. 

Thanks for your answer.

**Comment by patrikalienus** [comment]
Old versions of docs should _never_ be removed, imho.
I came here after taking over a project that is running version 9, och without the docs it'll be really hard to start the process of upgrading this old codebase.

**Reply by icyJoseph** [reply]
Not ideal, but some version of them still exists here https://github.com/vercel/next.js/tree/v9.5.6-canary.18/docs -
